tests/openebs-pool-container-failure_test.go

```diff
@@ -28,6 +29,9 @@ var (
 	clientSet         *chaosClient.LitmuschaosV1alpha1Client
 	err               error
 	containerIdBefore [9]string
+	chaosNamespace    = "litmus"
```

Comment:
you can keep a separate file defining these params (ns, labels etc.., ->> reuse)